examples/README.md

```diff
@@ -55,6 +53,8 @@ This page lists the available common switches with description. Unless stated ot
 | MLR | OT_MLR | Enables Multicast Listener Registration feature for Thread 1.2. |
 | MTD_NETDIAG | OT_MTD_NETDIAG | Enables the TMF network diagnostics on MTDs. |
 | MULTIPLE_INSTANCE | OT_MULTIPLE_INSTANCE | Enables multiple OpenThread instances. |
+| NAT64_BORDER_ROUTING | OT_NAT64_BORDER_ROUTING | Enables NAT64 border routing support for Border Router. |
+| NAT64_TRANSLATOR | OT_NAT64_TRANSLATOR | Enables NAT64 translator for Border Router. |
```

Comment:
it looks confusing to me to have two switches for nat64 feature.

Reply:
I think the scope of the two flags are relative small and have no intersections -- one for routing and one for translating, and we do supports external translator and external routing manager, the four possible combinations are all valid.